content/applications/inventory_and_mrp/barcode/operations/barcode_nomenclature.rst

```diff
@@ -1,84 +1,177 @@
+menu, and select :guilabel:`Default Nomenclature`.
+
+From this page, the list of *rules* and *barcode patterns* Odoo supports for the default
+nomenclature is accessible. The table contains all the information that can be condensed with a
+|UPC|/|EAN| barcode, along with the corresponding :guilabel:`Barcode Pattern`.
+
+.. _barcode/operations/use-upc-ean-barcodes:
+
+Use UPC/EAN barcodes in Odoo
+============================
+
+For product identification using |UPC|/|EAN| barcodes in Odoo, businesses obtain `barcodes
+<https://www.gs1us.org/upcs-barcodes-prefixes/how-to-get-a-upc-barcode>`_ purchased directly from
+GS1.
+
+The |UPC| and |EAN| barcode formats are predominantly used in their own regions. |UPC| is used only
+in the United States and Canada, while |EAN| is used everywhere else globally.
+
+A |UPC| is usually a 12-digit barcode used to identify most products, while |EAN| barcodes use
+13-digit barcodes to identify products.
+
+|UPC| codes can be converted to |EAN| by prefixing them with a zero. In Odoo, by default,
+|UPC|/|EAN| barcodes are automatically converted one way or another when trying to match a rule with
+the other encoding.
+
+Refer to the :ref:`default nomenclature list <barcode/operations/default-nomenclature-list>` to see
+a comprehensive list of all barcode patterns and rules to follow.
+
+Create rules
+------------
+
+|UPC| and |EAN| barcodes contain information in the barcode. Scanning these codes from the
+:ref:`default nomenclature list <barcode/operations/default-nomenclature-list>` automatically fills
+corresponding data in the Odoo database.
+
+Adding new barcode rules to this list ensures that non-standard (:dfn:`user-created`) formats are
+interpreted accurately.
+
+To create new rules, first activate :ref:`developer mode <developer-mode>`. Then, navigate to
+:menuselection:`Inventory app --> Configuration --> Barcode Nomenclatures`. Select the
+:guilabel:`Default Nomenclature` list item.
+
+On the :guilabel:`Default Nomenclature` page, select :guilabel:`Add a line` at the bottom of the
+table, which opens a window to create a new rule.
+
+.. image:: barcode_nomenclature/barcode-nomenclature-new-rule-popup.png
+   :align: center
+   :alt: Create Rules pop-up window on Default Nomenclature page.
+
+The :guilabel:`Rule Name` field is used internally to identify what the barcode represents. The
+barcode :guilabel:`Types` are different classifications of information that can be understood by the
+system (e.g., Package, Lot, Location, Coupon, etc.). The :guilabel:`Sequence` represents the
+priority of the rule; this means the smaller the value, the higher the rule appears on the table.
+
+Odoo follows the sequential order of this table, and will use the first rule it matches based on the
+sequence. The :guilabel:`Barcode Pattern` is how the sequence of letters or numbers is recognized by
+the system to contain information about the product.
```

Comment:
```suggestion
The :guilabel:`Barcode Pattern` represents how the sequence of letters or numbers is recognized by
the system to contain information about the product. Odoo follows the sequential order of this
table, and uses the first rule it matches, based on the sequence. 
```

Reply:
just to follow the order of things as they appear on the pop-up

Reply:
Suggestion implemented ✅